docs/src/user-guide/command-line-interface.md

```diff
@@ -382,71 +476,69 @@ npx eslint --max-warnings 10 file.js
 
 #### `-o`, `--output-file`
 
-Enable report to be written to a file.
+Write the output of linting results to a specified file.
 
-Example:
+* **Argument Type**: String. Path to file.
+* **Multiple Arguments**: No
+
+##### `-o`, `--output-file` example
 
 ```shell
 npx eslint -o ./test/test.html
 ```
 
-When specified, the given format is output into the provided file name.
-
 #### `-f`, `--format`
 
-This option specifies the output format for the console. Possible formats are:
+This option specifies the output format for the console.
 
-* [checkstyle](formatters/#checkstyle)
-* [compact](formatters/#compact)
-* [html](formatters/#html)
-* [jslint-xml](formatters/#jslint-xml)
-* [json](formatters/#json)
-* [junit](formatters/#junit)
-* [stylish](formatters/#stylish) (the default)
-* [tap](formatters/#tap)
-* [unix](formatters/#unix)
-* [visualstudio](formatters/#visualstudio)
+* **Argument Type**: String. One of the [built-in formatters](formatters/) or a custom formatter.
+* **Multiple Arguments**: No
+* **Default Value**: [`stylish`](formatters/#stylish)
 
-Example:
+If you are using a custom formatter defined in a local file, you can specify the path to the custom formatter file.
+
+An npm-installed formatter is resolved with or without `eslint-formatter-` prefix.
+
+When specified, the given format is output to the console. If you'd like to save that output into a file, you can do so on the command line like so:
 
 ```shell
-npx eslint -f compact file.js
+# Saves the output into the `results.txt` file.
+npx eslint -f compact file.js > results.txt
 ```
 
-You can also use a custom formatter from the command line by specifying a path to the custom formatter file.
+##### `-f`, `--format` example
 
-Example:
+Use the built-in `compact` formatter:
 
 ```shell
-npx eslint -f ./customformat.js file.js
+npx eslint --format compact file.js
 ```
 
-An npm-installed formatter is resolved with or without `eslint-formatter-` prefix.
+Use a local custom formatter:
+
+```shell
+npx eslint -f ./customformat.js file.js
+```
 
-Example:
+Use npm-installed formatter:
 
 ```shell
 npm install eslint-formatter-pretty
-
+# THEN
```

Comment:
I think this makes it read like you should do the first two lines OR the third, which is incorrect.

Reply:
i'll add some clarification comments to the code example

Reply:
added in 801544d8f8dfacfe3a27196d6eea56636917fa39